base/cvd/cuttlefish/package/BUILD.bazel

```diff
@@ -34,8 +34,11 @@ package_files(
         "cuttlefish-common/bin/libvk_swiftshader.so": "@swiftshader//:vk_swiftshader",
         "cuttlefish-common/bin/log_tee": "//cuttlefish/host/commands/log_tee",
         "cuttlefish-common/bin/logcat_receiver": "//cuttlefish/host/commands/logcat_receiver",
+        "cuttlefish-common/bin/lpadd": "//liblp:lpadd",
+        "cuttlefish-common/bin/lpmake": "//liblp:lpmake",
         "cuttlefish-common/bin/metrics_launcher": "//cuttlefish/host/commands/metrics_launcher",
         "cuttlefish-common/bin/metrics": "//cuttlefish/host/commands/metrics",
+        "cuttlefish-common/bin/mkbootimg.py": "@mkbootimg//:mkbootimg.py",
```

Comment:
Adding these targets with `@` to the packages list breaks the import to the internal repository. Consider not adding this here until a solution has been implemented internally. Example: b/413097319

Reply:
I see, but I have 5+ CLs WIP with this problem, I think it might be better to skip them on the internal repository and backfill, is that possible?

Reply:
sure, that's possible.

Just wondering if it made sense to do everything in this PR except adding the binary to the :cvd package as that's not going to be used until the marker file in android is updated, which would be blocked on the internal build anyways.

Reply:
Resolving the conversation and will work on the internal dependencies next week.